packages/wrangler/src/r2/helpers.ts

```diff
@@ -368,6 +376,20 @@ export function eventNotificationHeaders(
 	}
 	return headers;
 }
+
+export async function getEventNotificationConfig(
+	apiCredentials: ApiCredentials,
+	accountId: string,
+	bucketName: string
+): Promise<GetNotificationConfigResponse> {
+	const headers = eventNotificationHeaders(apiCredentials);
+	logger.log(`Fetching notification configuration for bucket ${bucketName}...`);
```

Comment:
In other commands you are sending metrics events, did you want to do that here too?

Reply:
Can you link to an example of what you mean?

Reply:
https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/r2/index.ts#L448-L450

Reply:
Not a blocker for this PR, and it is up to you whether to you want to add metrics for these commands.

Reply:
Oh, wow, nice, I did not know that was in there. That is actually a great idea but I'm going to make a separate work ticket to do it for all the notification handlers